src/table.ts

```diff
@@ -30,13 +30,29 @@ import {Mutation} from './mutation';
 import {Row} from './row';
 import {ChunkTransformer} from './chunktransformer';
 import {CallOptions} from 'google-gax';
+import {Bigtable, RequestCallback, OptionInterface} from '.';
+import {Instance} from './instance';
+import {google} from '../proto/bigtable';
 
 // See protos/google/rpc/code.proto
 // (4=DEADLINE_EXCEEDED, 10=ABORTED, 14=UNAVAILABLE)
 const RETRYABLE_STATUS_CODES = new Set([4, 10, 14]);
 // (1=CANCELLED)
 const IGNORED_STATUS_CODES = new Set([1]);
-
+export interface CreateFamilyOptions extends OptionInterface {
+  rule?: Rule;
+}
+export type CreateFamilyCallback = RequestCallback<
+  Family,
+  google.bigtable.v2.IFamily
```

Comment:
Shouldn't this be `google.bigtable.v2.ITable`?

Reply:
![image](https://user-images.githubusercontent.com/7764119/70451461-6833ae00-1a73-11ea-8917-1a7becc085c7.png)